### PR TITLE
Add sdk-components-react package

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-builder/issues/1129

Splitted base components into separate package. Exposed 3 entry points

- @webstudio-is/sdk-components-react
- @webstudio-is/sdk-components-react/metas
- @webstudio-is/sdk-components-react/props

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
- [ ] hi @istarkov, I need you to do
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/sdk-components-react/src/form.ws.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { FormIcon } from "@webstudio-is/icons/svg";
-import { form } from "../css/normalize";
 import {
   defaultStates,
   type PresetStyle,
   type WsComponentMeta,
   type WsComponentPropsMeta,
-} from "./component-meta";
+} from "@webstudio-is/react-sdk";
+import { form } from "@webstudio-is/react-sdk/css-normalize";
```
Comment (kof) [inline packages/sdk-components-react/src/form.ws.tsx:8]: are we going to keep react-sdk? I thought we will move react-sdk stuff that is framework agnostic to "sdk" or is this a separate step

**Review by kof — APPROVED** [review]

File: packages/sdk-components-react/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Webstudio
```
Comment (kof) [inline packages/sdk-components-react/LICENSE:3]: we need to update the date everywhere :)

File: packages/sdk-components-react/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Webstudio SDK
```
Comment (kof) [inline packages/sdk-components-react/README.md:1]:
```suggestion
# Webstudio SDK Components
```

File: packages/sdk-components-react/README.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+
+Webstudio SDK is a TypeScript API that lets you use your Webstudio site or some components in your custom codebase or just render a complete Remix Document.
+It is currently under development, but feel free to play with the the current [landing site](https://github.com/webstudio-is/webstudio-landing)
```
Comment (kof) [inline packages/sdk-components-react/README.md:4]: I think this was a copy-paste that doesn't apply to this package

File: packages/sdk-components-react/src/bold.ws.tsx [hunk 5]
```diff
@@ -4,9 +4,9 @@ import {
   type PresetStyle,
   type WsComponentMeta,
   type WsComponentPropsMeta,
-} from "./component-meta";
+} from "@webstudio-is/react-sdk";
+import { b } from "@webstudio-is/react-sdk/css-normalize";
```
Comment (kof) [inline packages/sdk-components-react/src/bold.ws.tsx:8]: isn't it going to treeshake if you just import it without a separate entry point?

File: packages/sdk-components-react/src/form.ws.tsx (hunk 1 above)
Comment (TrySound) [reply packages/sdk-components-react/src/form.ws.tsx:8]: Yep, here only components split

File: packages/sdk-components-react/src/bold.ws.tsx (hunk 5 above)
Comment (TrySound) [reply packages/sdk-components-react/src/bold.ws.tsx:8]: Semantically not good to export tag styles from sdk without namespace. Though namespace export or object will kill treeshaking.

**Review by istarkov — APPROVED** [review]

**Pull request merged**